zerver/lib/actions.py

```diff
@@ -2385,6 +2385,17 @@ def fetch_stream_subscriber_user_ids(stream: Stream) -> List[int]:
             [(sub.user_profile, stream) for (sub, stream) in subs_to_activate],
             already_subscribed)
 
+def get_available_notification_sounds() -> Iterable[str]:
+    notification_sounds_path = os.path.join(
+        os.path.dirname(__file__), '../../static/audio/notification_sounds')
```

Comment:
I don't think we want to construct this path relative to the location of `actions.py`, but I'm not sure what the right way would be. Ignore my comment for now.